pandas/tests/groupby/methods/test_kurt.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+import pandas as pd
+import pandas._testing as tm
+
+
+def test_groupby_kurt_equivalence():
+    # Test that that groupby kurt method (which uses libgroupby.group_kurt)
+    #  matches the results of operating group-by-group (which uses nanops.nankurt)
+    nrows = 1000
+    ngroups = 3
+    ncols = 2
+    nan_frac = 0.05
+
+    arr = np.random.default_rng(2).standard_normal((nrows, ncols))
+    arr[np.random.default_rng(2).random(nrows) < nan_frac] = np.nan
+
+    df = pd.DataFrame(arr)
+    grps = np.random.default_rng(2).integers(0, ngroups, size=nrows)
+    gb = df.groupby(grps)
+
+    result = gb.kurt()
+
+    grpwise = [grp.kurt().to_frame(i).T for i, grp in gb]
+    expected = pd.concat(grpwise, axis=0)
+    expected.index = expected.index.astype(result.index.dtype)  # 32bit builds
```

Comment:
Can you use `"intp"` here instead of having the expected depend on the result.

Reply:
Fixed

Reply:
The test seems to fail in some environments after I changed it to `"intp"`. Do I need to separately assert the index and the values like some other tests do?

Reply:
Sorry, should be int64 I think.